### PR TITLE
🎮 Feature/console hooks

### PR DESCRIPTION
**Pull request by romanesko**

### CODE REVIEW

File: twake/backend/node/src/cli/cmds/generate_cmds/users-and-companies.ts [hunk 1]
```diff
@@ -57,7 +57,7 @@ const command: yargs.CommandModule<{}, CLIArgs> = {
       console.log("Creating user", userInCompany);
       const created = await userService.users.create(getUserInstance(userInCompany.user));
 
-      return userService.companies.addUserInCompany(userInCompany.company, created.entity);
+      return userService.companies.setUserRole(userInCompany.company.id, created.entity.id);
```
Comment (RomaricMourgues) [inline twake/backend/node/src/cli/cmds/generate_cmds/users-and-companies.ts:60]: Can we remove a user from a company with this `setUserRole` method ?

File: twake/backend/node/src/core/platform/services/database/services/orm/utils.ts [hunk 2]
```diff
@@ -73,7 +81,7 @@ export function secureOperators<Entity>(
   return findOptions;
 }
 
-let microCounter = 0; //We do not really use nanotime, our goal is to ensure unicity first.
+const microCounter = 0; //We do not really use nanotime, our goal is to ensure unicity first.
```
Comment (RomaricMourgues) [inline twake/backend/node/src/core/platform/services/database/services/orm/utils.ts:84]: I don't think this variable is used anymore is it ? If not you can remove it

**Review by RomaricMourgues — COMMENTED** [review]
It seems great, we will soon test it on QA and see if the hooks are working well.

Question:
- Did we move this two endpoints to node already? https://github.com/linagora/Twake/blob/main/twake/backend/core/src/Twake/Users/Controller/Adapters/Console/Api.php

File: twake/backend/node/src/cli/cmds/generate_cmds/users-and-companies.ts (hunk 1 above)
Comment (romanesko) [reply twake/backend/node/src/cli/cmds/generate_cmds/users-and-companies.ts:60]: We have `removeUserFromCompany` function for that

Comment (RomaricMourgues) [reply twake/backend/node/src/cli/cmds/generate_cmds/users-and-companies.ts:60]: So "setUserRole" automatically add the user in the company if it doesn't exists ?

Comment (romanesko) [reply twake/backend/node/src/cli/cmds/generate_cmds/users-and-companies.ts:60]: yes it is. 
I can rename it to get away from ambiguity

Comment (RomaricMourgues) [reply twake/backend/node/src/cli/cmds/generate_cmds/users-and-companies.ts:60]: No it's great don't change it :)

**Pull request merged**